website/static/js/signIn.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Sign in view model used for login components
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout-validation').init({insertMessages: false});  // override default DOM insertions
+var $ = require('jquery');
+
+var $osf = require('osfHelpers');
```

Comment:
For forwards-compatibility with https://github.com/CenterForOpenScience/osf.io/issues/2143, let's use `require('js/osfHelpers')` from now on.